dmcontent/questions.py

```diff
@@ -459,6 +459,7 @@ class Pricing(Question):
     def __init__(self, data, *args, **kwargs):
         super(Pricing, self).__init__(data, *args, **kwargs)
         self.fields = data['fields']
+        self.decimal_place_restriction = True if data.get('decimal_place_restriction') else False
```

Comment:
Is this cleaner just as `data.get('decimal_price_restriction', False)`?

Reply:
Yes, good point. Will change.